packages/block-editor/src/components/collection-manager/index.js

```diff
@@ -0,0 +1,59 @@
+import { ToggleGroup } from "../index";
+import {
+  collectionToggles,
+  heroToggles,
+  carouselToggles,
+  cardToggles
+} from "./toggles";
+
+const CollectionManager = (props) => {
```

Comment:
We should rather keep this component in the `with-elements-visibility` folder since it is just a wrapper over the ToggleGroup component strictly used by that specific filter.